### PR TITLE
Backfill invocation uuid take 2

### PR DESCRIPTION
**Pull request by luluz66**
Update invocations and target statuses in batches

### CODE REVIEW

File: server/tables/tables.go [hunk 1]
```diff
@@ -578,78 +578,73 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
...
 	return postMigrate, nil
 }
 
-func backfillInvocationUUIDIfNull(db *gorm.DB) error {
-	if err := db.Exec(`
-		UPDATE Invocations SET invocation_uuid = UUID_TO_BIN(invocation_id) 
-		WHERE invocation_uuid IS NULL
-	`).Error; err != nil {
-		return err
+func updateInBatch(db *gorm.DB, baseQuery string, batchSize int64) error {
```
Comment (vadimberezniker) [inline server/tables/tables.go:598]: nit: let's call this updateInBatches since there will be multiple batches

File: server/tables/tables.go [hunk 2]
```diff
@@ -578,78 +578,73 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
...
-			(SELECT invocation_uuid FROM Invocations i 
-			WHERE i.invocation_pk=ts.invocation_pk)
-		WHERE invocation_uuid IS NULL
-	`).Error
 }
 
 func postMigrateInvocationUUIDForMySQL(db *gorm.DB) error {
-	if err := db.Exec(`UPDATE Invocations SET invocation_uuid = UUID_TO_BIN(invocation_id)`).Error; err != nil {
+	updateInvocationStmt := `UPDATE Invocations SET invocation_uuid = UNHEX(REPLACE(invocation_id, "-","")) WHERE invocation_uuid IS NULL`
+	err := updateInBatch(db, updateInvocationStmt, 10000)
```
Comment (vadimberezniker) [inline server/tables/tables.go:615]: nit: inline the updateInBatch call into the if statement below

File: server/tables/tables.go [hunk 3]
```diff
@@ -578,78 +578,73 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
...
-			invocationUUID, err := uuid.StringToBytes(invocation.InvocationID)
-			if err != nil {
-				return err
-			}
-			if err := db.Exec(`UPDATE TargetStatuses SET invocation_uuid = ? WHERE invocation_pk = ? `, invocationUUID, invocation.InvocationPK).Error; err != nil {
-				return err
-			}
-			if err := db.Exec(`UPDATE Invocations SET invocation_uuid = ? WHERE invocation_id = ? `, invocationUUID, invocation.InvocationID).Error; err != nil {
-				return err
+	res := db.Select("invocation_id", "invocation_pk").
```
Comment (vadimberezniker) [inline server/tables/tables.go:631]: nit: these line breaks are a little awkward

you could store the fn into a variable to reduce the lenth of the line

something like 
updateFn := func(tx *gorm.DB, batch int) error { ... }
res := db.Select("invocation_id", "invocation_pk").FindInBatches(&results, 100, updatefn)

**Review by vadimberezniker — APPROVED** [review]
LGTM, only minor nits

File: server/tables/tables.go (hunk 3 above)
Comment (luluz66) [reply server/tables/tables.go]: Done

File: server/tables/tables.go (hunk 2 above)
Comment (luluz66) [reply server/tables/tables.go]: Done

File: server/tables/tables.go (hunk 1 above)
Comment (luluz66) [reply server/tables/tables.go]: Done

**Pull request merged**